gallery/src/components/NxSkeletonLoader/NxSkeletonLoaderPage.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { faEdit, faWalking } from '@fortawesome/free-solid-svg-icons';
+import { NxFontAwesomeIcon, NxH2, NxP, NxSkeletonLoader, NxSpan, NxTable, NxThreatCounter, NxThreatIndicator, NxTile }
+  from '@sonatype/react-shared-components';
+import React, { useEffect, useState } from 'react';
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+
+export default function NxSkeletonLoaderPage() {
+  const [showSecondTile, setShowSecondTile] = useState(false);
+
+  useEffect(function() {
+    setTimeout(() => { setShowSecondTile(true); }, 800);
```

Comment:
The purpose of the second tile is to prove that the skeleton animations are synchronized even for elements that render at different times